src/modules/recipe/contactListItem/__tests__/contactListItem.test.js

```diff
@@ -33,4 +33,25 @@ describe('recipe-contact-list-item', () => {
         const nameEl = element.shadowRoot.querySelector('p');
         expect(nameEl.textContent).toBe(CONTACT.Name);
     });
+
+    it('fires select event when clicked', () => {
```

Comment:
I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

Reply:
done.